packages/rolldown/tests/fixtures/builtin-plugin/alias/at-alias-prefix/_config.ts

```diff
@@ -2,6 +2,7 @@ import { aliasPlugin } from 'rolldown/experimental'
 import { defineTest } from '@tests'
 
 export default defineTest({
+  skip: true,
```

Comment:
This test failed in this PR, which considered expected.

```
aliasPlugin({
        entries: [
          { find: '@src', replacement: 'src' },
          { find: '@utils', replacement: './utils' },
        ],
      }),
```

would transform

```
import '@src/index.js'
```

to 

```
import 'src/index.js'
```

`import 'src/index.js'` is not handleable with default configuration of rolldown.

@IWANABETHATGUY cc

Reply:
I'll merge this PR first. If we should make `{ find: '@src', replacement: 'src' },` work by default in rolldown, let's have a discussion then.